plugin.rb

```diff
@@ -12,6 +12,4 @@
 
 enabled_site_setting :discourse_login_client_enabled
 
-auth_provider title_setting: "discourse_login_client_button_title",
-              icon: "fab-discourse",
-              authenticator: DiscourseLoginClientAuthenticator.new
+auth_provider icon: "fab-discourse", authenticator: DiscourseLoginClientAuthenticator.new
```

Comment:
Removed the `title_setting`, this will fallback to `discourse_login.title` which is cleaner anyway.